src/output.c

```diff
@@ -230,7 +243,7 @@ output_update_for_layout_change(struct server *server)
 
 	/*
 	 * "Move" each wlr_output_cursor (in per-output coordinates) to
-	 * align with the seat cursor.  Set a default cursor image so
+	 * align with the seat cursor.	Set a default cursor image so
```

Comment:
This seems to replace a double space with a tab, should be replaced by a single space instead.

Reply:
Fixed by replacing with a single space.